app/views/agent/capturePreference.scala.html

```diff
@@ -22,12 +22,10 @@
 
 @(preferenceForm: Form[PreferenceModel])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
-@main_template(messages("capturePref.title")) {
+  @main_template(if(preferenceForm.errors.nonEmpty) "Error: " + messages("capturePref.title") else messages("capturePref.title")) {
```

Comment:
"Error: " will need to go in a message as it will need to be translated to Welsh